### PR TITLE
fix bug rewards == null

### PR DESCRIPTION
**Pull request by ndxbinh1922001**

### CODE REVIEW

**Review by LVH-Tony — COMMENTED** [review]
Anh thấy cách sửa của em đã hiệu quả trong việc ngăn normalize tensor rỗng trước khi thực hiện lệnh dưới. Tuy nhiên, anh nghĩ an toàn hơn là em nên vẫn tạo `self.rewards_normalized` ngay cả khi `self.rewards` là rỗng. Cũng có thể là nên biến `self.rewards_normalized` thành `torch.zeros_like(self.rewards)` trong trường hợp tensor rỗng. Em thấy sao

File: einstein/rewards/reward.py
Comment (LVH-Tony) [inline einstein/rewards/reward.py:1]: I think this fix is ok to prevent normalization operations on an empty tensor before proceeding but it would be beneficial to also initialize `self.rewards_normalized` even when `self.rewards` is empty.
I suggest you should still set `self.rewards_normalized` variable to something maybe like `torch.zeros_like(self.rewards)` in the case of empty tensors to avoid any future bugs for not having a `self.rewards_normalized`

**Review by LVH-Tony — APPROVED** [review]
